content/blog/2015-07_Meet Nukomeeter  Adam.md

```diff
@@ -0,0 +1,41 @@
+---
+created_at: 2015-07-02
+kind: article
+publish: true
+title: "Meet Nukomeeter: Adam"
+authors:
+- adam
+- kasia
+tags: 
+- interview
```

Comment:
Improved

*Kasia Popek*

Communications Manager at Nukomeet


___


You can contact me at

mail *kasia@nukomeet.com*

mobile *+48 609 160 095*


Visit our website *www.nukomeet.com*




On Thu, Jul 9, 2015, at 10:13 PM, Bartłomiej Wiaduch wrote:
> In content/blog/2015-07_Meet Nukomeeter  Adam.md[1]:


> > @@ -0,0 +1,41 @@
> +--- +created_at: 2015-07-02 +kind: article +publish: true +title:
> "Meet Nukomeeter: Adam" +authors: +- adam +- kasia +tags: +- interview
> @myviridis[2] possible tags are ['business', 'programming',
> 'innovation', 'society', 'technology', 'web', 'meetup']


> they are defined in
> https://github.com/nukomeet/nukomeet.com/blob/master/Rules


> — Reply to this email directly or view it on GitHub[3].


>


Links:

  1. https://github.com/nukomeet/nukomeet.com/pull/192#discussion_r34298163
  2. https://github.com/myviridis
  3. https://github.com/nukomeet/nukomeet.com/pull/192/files#r34298163